pallets/registrar/src/benchmark_blob.rs

```diff
@@ -33,54 +33,53 @@ pub struct BenchmarkBlob {
-    153, 57, 69, 212, 166, 142, 68, 212, 239, 160, 6, 119, 200, 95, 12, 230, 120, 121, 157, 62,
-    255, 2, 66, 83, 185, 14, 132, 146, 124, 198, 128, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-    0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 153, 1, 128, 11, 0, 128, 21, 69, 87, 39, 80,
-    189, 63, 244, 210, 244, 71, 86, 224, 172, 98, 5, 171, 30, 40, 194, 211, 222, 117, 47, 81, 150,
-    51, 191, 236, 129, 237, 184, 128, 223, 91, 79, 3, 86, 79, 50, 130, 149, 40, 193, 33, 8, 158,
-    35, 82, 137, 134, 195, 10, 60, 248, 193, 39, 236, 182, 225, 143, 110, 206, 62, 80, 128, 77,
-    174, 114, 63, 107, 0, 39, 241, 147, 41, 150, 41, 130, 189, 148, 216, 5, 24, 47, 167, 29, 117,
-    8, 120, 210, 51, 35, 145, 27, 94, 189, 250, 169, 1, 159, 12, 182, 243, 110, 2, 122, 187, 32,
-    145, 207, 181, 17, 10, 181, 8, 127, 137, 0, 104, 95, 6, 21, 91, 60, 217, 168, 201, 229, 233,
-    162, 63, 213, 220, 19, 165, 237, 32, 0, 0, 0, 0, 0, 0, 0, 0, 104, 95, 8, 49, 108, 191, 143,
-    160, 218, 130, 42, 32, 172, 28, 85, 191, 27, 227, 32, 0, 0, 0, 0, 0, 0, 0, 0, 128, 254, 108,
-    203, 37, 75, 132, 240, 210, 32, 46, 181, 5, 189, 223, 159, 84, 203, 158, 189, 15, 178, 113,
-    144, 114, 233, 46, 229, 124, 29, 161, 216, 9, 190, 247, 74, 173, 54, 61, 117, 185, 65, 198,
-    150, 80, 9, 116, 166, 1, 185, 33, 23, 38, 14, 102, 24, 248, 23, 224, 15, 137, 66, 208, 101,
-    122, 20, 176, 0, 0, 32, 0, 0, 0, 16, 0, 8, 0, 0, 0, 0, 4, 0, 0, 0, 1, 0, 0, 5, 0, 0, 0, 5, 0,
-    0, 0, 6, 0, 0, 0, 6, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 9, 1, 63, 32, 6, 222, 61, 138, 84, 210,
-    126, 68, 169, 213, 206, 24, 150, 24, 242, 45, 180, 180, 157, 149, 50, 13, 144, 33, 153, 76,
-    133, 15, 37, 184, 227, 133, 236, 219, 232, 203, 100, 143, 101, 214, 239, 232, 53, 250, 27, 50,
-    253, 149, 140, 97, 244, 92, 63, 91, 253, 137, 187, 12, 221, 174, 199, 161, 103, 210, 200, 95,
-    12, 230, 120, 121, 157, 62, 255, 2, 66, 83, 185, 14, 132, 146, 124, 198, 128, 0, 0, 0, 0, 0, 0,
-    0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 21, 1, 128, 3, 0,
-    128, 21, 69, 87, 39, 80, 189, 63, 244, 210, 244, 71, 86, 224, 172, 98, 5, 171, 30, 40, 194,
-    211, 222, 117, 47, 81, 150, 51, 191, 236, 129, 237, 184, 128, 223, 91, 79, 3, 86, 79, 50, 130,
-    149, 40, 193, 33, 8, 158, 35, 82, 137, 134, 195, 10, 60, 248, 193, 39, 236, 182, 225, 143, 110,
-    206, 62, 80, 169, 1, 159, 12, 182, 243, 110, 2, 122, 187, 32, 145, 207, 181, 17, 10, 181, 8,
-    127, 137, 0, 104, 95, 6, 21, 91, 60, 217, 168, 201, 229, 233, 162, 63, 213, 220, 19, 165, 237,
-    32, 0, 0, 0, 0, 0, 0, 0, 0, 104, 95, 8, 49, 108, 191, 143, 160, 218, 130, 42, 32, 172, 28, 85,
-    191, 27, 227, 32, 0, 0, 0, 0, 0, 0, 0, 0, 128, 254, 108, 203, 37, 75, 132, 240, 210, 32, 46,
-    181, 5, 189, 223, 159, 84, 203, 158, 189, 15, 178, 113, 144, 114, 233, 46, 229, 124, 29, 161,
-    216, 9,
+    0, 55, 133, 53, 79, 12, 10, 168, 38, 54, 66, 50, 252, 54, 227, 253, 113, 215, 29, 209, 8, 221,
```

Comment:
This has changed because previously I used `let account = ALICE;` below, but that doesn't work because of an invalid signature error when running:

```
cargo test -p pallet-registrar --features=runtime-benchmarks
```

I didn't notice before because this command is not being run in CI